llorens/day05/Day05Solver.py

```diff
@@ -0,0 +1,66 @@
+def solve_part_1(sequence):
+
+    sequence = sequence.splitlines()
+
+    pos = 0
+    jumps = 0
+
+    while 0 <= pos < len(sequence):
+        jumps += 1
+        displacement = int(sequence[pos])
+        sequence[pos] = int(sequence[pos]) +1
+        pos += displacement
```

Comment:
It took me a little to understand this because of the int casting. Anyway, it is simple enough 👍